appcues/src/main/java/com/appcues/debugger/DebugMode.kt

```diff
@@ -0,0 +1,6 @@
+package com.appcues.debugger
+
+internal sealed class DebugMode {
+    data class Debugger(val deepLinkPath: String?) : DebugMode()
+    object ScreenCapture : DebugMode()
```

Comment:
are we thinking that when we have a "token" from the deep link we will be passing that here?

Reply:
I _think_ so. There is a separate ticket for the API integration once that's ready, this is focused on the UI mechanics mostly. But good point to think about there.